lib/prx_auth/rails/ext/controller.rb

```diff
@@ -4,16 +4,42 @@ module PrxAuth
   module Rails
     module Controller
       def prx_auth_token
+        rack_auth_token = env_prx_auth_token
+        return rack_auth_token if rack_auth_token.present?
```

Comment:
Okay, so this must be where it prefers HTTP Authorization header over your session.